closure/bin/build/closurebuilder.py

```diff
@@ -200,9 +201,14 @@ def main():
 
   # Make our output pipe.
   if options.output_file:
-    out = open(options.output_file, 'w')
+    out = io.open(options.output_file, 'wb')
   else:
-    out = sys.stdout
+    version = sys.version_info[:2]
+    if version >= (3,0):
```

Comment:
At a glance, this change seems fine to me. There's a bigger underlying question here of python 3 support in general for our tools. This may fix this one issue, but I'm sure there are other places in the code that aren't Py3 compliant. Also, I don't have a lot of experience making scripts both Py3 and Py2 compatible, but I'd hate to see this type of pattern repeated all over the place (i.e., `if py3: do blah else: do foo`).

Let me chat with some other folks about this.